sphinx/testing/fixtures.py

```diff
@@ -118,10 +118,10 @@ def app_params(
     # always write to the temporary directory
     kwargs.setdefault('builddir', srcdir / '_build')
 
-    return _app_params(args, kwargs)
+    return AppParams(args, kwargs)
 
 
-_app_params = namedtuple('_app_params', 'args,kwargs')  # NoQA: PYI024
+AppParams = namedtuple('AppParams', 'args,kwargs')  # NoQA: PYI024
```

Comment:
Can we keep this as a private name?

Reply:
I made it public because we are importing it from another file. Do you still want it to be a private name given that?

Reply:
Yes please -- we're fine to use private names within the package, but I don't want to expose it as public for third-party users.

A

Reply:
Ok I won't change the name

Reply:
I have reverted the name @AA-Turner